index.php

```diff
@@ -14,11 +15,16 @@
 <center>
     <form method="GET" class="form-horizontal">
         <fieldset>
-            <legend>Please type your <a href="https://beta.wikiversity.org/wiki/List_of_ISO_639-1_codes">language code in ISO 639-1</a>. For example: pl for Polish, de for German</legend>
+            <legend>Please choose your language from the list below. </legend>
             <div class="form-group">
-                <label class="col-md-4 control-label" for="lang">Language code:</label>
+                <label class="col-md-4 control-label" for="lang">Language:</label>
                 <div class="col-md-4">
-                    <input type="text" value="<?php echo isset($_SESSION['lang']) ? $_SESSION['lang'] : '' ?>" id="lang" name="lang" class="form-control input-md" required="required" autofocus="autofocus" pattern="[A-Za-z]{2}" title="Two letter language code"/>
+                    <select id="lang" name="lang" class="form-control">
+                        <option value="" selected disabled hidden>Choose here</option>
```

Comment:
Delete `selected` and use it in `foreach` loop - depending on `$_SESSION['lang']`